src/main/java/weiss/wordle/WordleDictionary.java

```diff
@@ -0,0 +1,55 @@
+package weiss.wordle;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class WordleDictionary {
+    //ArrayList to store all the lines in the dictionary
+    ArrayList<String> dictionary = new ArrayList<String>();
+    String word;
+    String definition;
+    String[] dictLine;
+    ArrayList<String> justWords = new ArrayList<String>();
```

Comment:
member variables should be private

Reply:
String word;
String definition;
String[] dictLine;

These should be local variables (defined in the method), not member variables.